src/sv_primlit_integral.rs

```diff
@@ -138,6 +153,34 @@ impl SvPrimaryLiteralIntegral {
         ret
     }
 
+    /* Returns whether the MSB of data_01 is high. The size must be correctly specified. */
+    pub fn data_01_msb_high(&self) -> bool {
```

Comment:
Would you rename this something like `data_01_msb_high` -> `is_set_msb_01`, and similarly `data_xz_msb_high` -> `is_set_msb_xz`?

Reply:
I'm updating it immediately.